### DISCUSSION TITLE
MultiSelect: Add `onRemove` callback function, the same as in the Chip component

### DISCUSSION THREAD

**Discussion by datacampsis** [primefaces/community#364] [PrimeReact] [answered]
In the Chip component, you can call the function `onRemove` to get the removed value.
In MultiSelect component, using `display="chip"` does not have this function, and it becomes necessary in some cases.

**Comment by melloware** [comment]
Open a GitHub issue ticket.

**Comment by GabriellDatacamp** [comment, marked as answer]
Ok, i open the issue : `#5247`
